Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml

```diff
@@ -36,7 +49,7 @@
     walkModifier: 0.4
     sprintModifier: 0.6
   - type: Clothing
-    size: 50
+    size: 101
```

Comment:
Should these be like 120? I thought syndie duffels stored more than that.

Reply:
Basic duffles store 120 and I thought that'd be fine for hardsuits but I guess you're right. Not sure if syndie one holds more will check.

Reply:
Just checked all duffels are 120 so this lets people fit a hardsuit, mask and oxy.